Description: change data path
 Relocate non-binary architecture-specific files to /usr/share to ensure correct installation
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-12-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.c
+++ b/main.c
@@ -102,7 +102,7 @@
 
 void load_fragment_shader() {
   system("mkdir -p gen && unifdef -b -x2 -DDEBUG -o gen/fshader-debug.glsl fshader.glsl");
-  load_shader(fragment_shader, "gen/fshader-debug.glsl", GL_FRAGMENT_SHADER);
+  load_shader(fragment_shader, "/usr/share/eternalforest-by-soulman/fshader-debug.glsl", GL_FRAGMENT_SHADER);
 }
 
 void fragment_shader_changed(GFileMonitor * file_monitor, GFile *file, GFile *other_file, GFileMonitorEvent event_type, GtkGLArea * area) {
